### PR TITLE
Issue #9929: Ensure Suns Java Style Checkstyle Coverage

### PR DESCRIPTION
**Pull request by Ritik2604**
fixes Issue #9929  Ensure sun java checkstyle coverage

Mapped CommentsIndentationCheck to Sun style Block Comment

### CODE REVIEW

**Comment by Ritik2604** [comment]
@romani  review needed

**Comment by rnveach** [comment]
GitHub, rebase

**Comment by rnveach** [comment]
@Ritik2604 Are you still willing to get this merged? If so, please start by rebasing this on the latest master.

**Comment by pbludov** [comment]
GitHub, generate site

File: src/xdocs/sun_style.xml [hunk 1]
```diff
@@ -366,8 +366,23 @@
...
                   </a>
                 </td>
-                <td>???</td>
-                <td/>
+                <td>
+                  <span class="wrapper inline">
+                    <img src="images/ok_blue.png" alt=""/>
+                  </span>
+                  <a href="config_misc.html#CommentsIndentation">
+                    CommentsIndentation
```
Comment (romani) [inline src/xdocs/sun_style.xml:374]: We need to put comment here that we do not cover
>```
block comment should be preceded by a blank line to set it apart from the rest of the code.

/*
 * Here is a block comment.
 */
```
It should be done by some other Chek

File: src/it/resources/com/sun/checkstyle/test/chapter5comments/rule511blockcomments/InputCommentsIndentationSurroundingCode.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+import java.util.Arrays;
+
+// some
+public class InputCommentsIndentationSurroundingCode
+{
+    private void foo1() {
+        if (true) {
+            // here initialize some variables
+            int k = 0; // trailing comment
```
Comment (romani) [inline src/it/resources/com/sun/checkstyle/test/chapter5comments/rule511blockcomments/InputCommentsIndentationSurroundingCode.java:12]: I do not see a reason to copy all that inputs better to use few and focus on that matter for sun style description.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/sun_style.xml (hunk 1 above)
Comment (romani) [reply src/xdocs/sun_style.xml:374]: We do not have check that demand empty line in block comment :)

**Comment by rnveach** [comment]
@Ritik2604 ping

**Comment by nick-mancuso** [comment]
We have lost contact with author, anyone is welcome to reuse code to solve related issue.

**Pull request closed without merging**